### DISCUSSION TITLE
Evolve chip (or tag) to include navigation

### DISCUSSION THREAD

**Discussion by lauraocana** [Telefonica/mistica-design#613] [Proposals]
Lanzo aquí esta discusión que tuvimos por otro canal con @yceballost y @HectorSancho 
Estuvimos valorando la posibilidad de evolucionar el chip de Mística para incluir el caso en que pudiera ser navegable. También vimos la posibilidad de que pudiera ser la tag, pero en mi opinión sería mejor reservarla para marcar estados y mantenerla exenta de interacción. 

![git](https://user-images.githubusercontent.com/68842873/178520446-dc725b02-e0c4-47a2-bee5-e3f410adc0d6.png)
![jira](https://user-images.githubusercontent.com/68842873/178520478-4284d2c4-a110-4b9e-af85-f5a23ddc2643.png)

**Comment by AnaMontes11** [comment]
Tras hablar conjuntamente, se decide ampliar la definición de las chips para incluir navegación:
https://github.com/Telefonica/webapp/pull/1907/files


Se crea tarea para actualizar las specs de las chips añadiendo esta nueva definición.

**Reply by aweell** [reply]
Archivo discussion 👍🏼

**Comment by aweell** [comment]
https://github.com/Telefonica/mistica-design/issues/621
